### ISSUE TITLE
[BUG] cluster conditions sais "WaitApplied(2) [Bundle fleet-agent-local] "

### ISSUE THREAD

**Issue by colaH16** [rancher/rancher#42261]
**It's Aarch64. user Iaas on Oracle cloud infrastructure** 

**Rancher Server Setup**
- Rancher version: 
- Installation option (Docker install/Helm Chart): `Helm Chart`
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): k3s
- Proxy/Cert Details: `self signed cert. `

**Information about the Cluster**
- Kubernetes version: `v1.26.6 +k3s1`
- Cluster Type (Local/Downstream): `Local`
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom): `Admin`
  - If custom, define the set of permissions:

**Describe the bug**

`Fleet agent pod Failed to register agent.`

**To Reproduce**

on aarch64 machine
install k3s, install rancher with helm.
create gitea pod, connect gitea to rancher continuous delivery.
1. install k3s `curl -sfL https://get.k3s.io/ | K3S_KUBECONFIG_MODE="660" INSTALL_K3S_VERSION=v1.26.6+k3s1 K3S_TOKEN=rancher.oci.cola16 sh -s - server --cluster-init`
2. install cert-manager 
```
helm install cert-manager jetstack/cert-manager \
  --namespace cert-manager \
  --create-namespace \
  --set installCRDs=true
```
3. install rancher
```
helm install rancher rancher-latest/rancher \
  --namespace cattle-system \
  --set hostname=<domain> \
  --set bootstrapPassword=admin
```
4. create gitea pod, connect gitea to rancher continuous delivery.

**Result**
`fleet-agent-local` bundle, `gitea-svc` bundle stuck in wait applied

**Expected Result**

The pod described in `gitea` is created.

**Screenshots**

![Screenshot 2023-07-29 at 10 30 14 PM](https://github.com/rancher/rancher/assets/15865211/a9ca6b70-69e2-4721-9e34-d28119db5420)
![Screenshot 2023-07-29 at 10 29 16 PM](https://github.com/rancher/rancher/assets/15865211/e6b5479c-ef11-400e-ab87-a7c756df47ad)
![Screenshot 2023-07-29 at 10 29 38 PM](https://github.com/rancher/rancher/assets/15865211/9dfceff7-4b3d-48be-aa3e-9e3458972513)
![Screenshot 2023-07-29 at 10 29 40 PM](https://github.com/rancher/rancher/assets/15865211/b3e33d45-5990-4edd-aa9c-c6896e151f29)
![Screenshot 2023-07-29 at 10 29 29 PM](https://github.com/rancher/rancher/assets/15865211/24f0b4a2-8ba4-434b-bcfe-a00c7137230b)

**Additional context**

**Comment by colaH16** [comment]
I found a secret called fleet-agent-bootstrap.
The apiServerURL was a public URL.
The public URL is that only access by logging in with SSO.

So I changed it to a private URL  that pointing LoadBalancer.
But I got an unauthorized error and couldn't proceed.

I tried applying rancher.cattle-system.svc.cluster.local, but I got an X509 error.

What should be the value of CAcert, what should be the value of apiServer, 
How can I verify that this is normal?

Translated with www.DeepL.com/Translator (free version)
